src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final int maxParameterValue = 50;
+    private static final int maxFigureNumber = 5;
+    private static final Random random = new Random();

```

Comment:
Don't forget how we name our constants. Applies to all your fields. See [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)